batavia/builtins/repr.js

```diff
@@ -7,7 +7,7 @@ function repr(args, kwargs) {
     if (kwargs && Object.keys(kwargs).length > 0) {
         throw new exceptions.TypeError.$pyclass("repr() doesn't accept keyword arguments")
     }
-    if (!args || args.length !== 1) {
+    if (args === undefined || args.length !== 1) {
```

Comment:
I'm actually not sure why we are testing args at all here. Line 4 already takes the length, making the assumption that it is defined, so by this point, both `!args` and `args === undefined` will be false.

Reply:
!args failed a test where args was an empty string. I'll take a look.

Reply:
Oh. builtins.repr() should always be called with an array for the args param.

Reply:
Reverted this to the original. Could still be undefined, for example:

var arr = buggyarraybuilder() // returns undefined because javascript
builtins.repr(arr, {})

arguments length still 2, but args is undefined.

Reply:
I have a vague plan to provide a single validation method to use for all the places where we currently validate "batavia calling conventions". Confirming both that this first argument is not undefined would be a good addition to that in general.